WPMedia/fusion-news-theme-blocks
# TMEDIA-150 - Add bandito support to promos

## Description

Add the bandito registerSuccessEvent to PromoHeadline and PromoImage component, and update all components to utilize the shared Headline and Image components

## Jira Ticket
- [TMEDIA-150](https://arcpublishing.atlassian.net/browse/TMEDIA-150)

## Acceptance Criteria
1. Customers can run Bandito tests on the following Arc Blocks: 
    * XL Promo Block
    * L Promo Block
    * Medium Promo Block
    * Small Promo Block
    * XL Manual Promo Block
    * L Manual Promo Block
    * Medium Manual Promo Block
    * Small Manual Promo Block

## Test Steps

1. Checkout this branch `git checkout TMEDIA-150-add-banito-to-promos`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/extra-large-promo-block,@wpmedia/large-promo-block,@wpmedia/medium-promo-block,@wpmedia/small-promo-block,@wpmedia/extra-large-manual-promo-block,@wpmedia/large-manual-promo-block,@wpmedia/medium-manual-promo-block,@wpmedia/small-manual-promo-block,@wpmedia/shared-styles`
3.  Verify promo blocks didn't change - test page - http://localhost/pf/test-all-promos/?_website=the-gazette

## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._

## Reviewer Checklist
_The reviewer of the PR should copy-paste this template into the review comments on review._
